test/levels.test.js

```diff
@@ -0,0 +1,92 @@
+import _ from 'lodash';
+import levels from '../src/levels';
+
+describe('levels', () => {
+  it('should be defined', () => {
+    let expectedLevels = [
+      'alert',
+      'critical',
+      'debug',
+      'emergency',
+      'error',
+      'fatal',
+      'info',
+      'informational',
+      'notice',
+      'silly',
+      'trace',
+      'verbose',
+      'warn',
+      'warning'];
+
+    _.forEach(expectedLevels, (lvl) => {
+      expect(levels[lvl]).toBeDefined();
+    });
+  });
+});
+
+describe('levelToLevelName', () => {
+  it('should be defined', () => {
+    expect(levels.levelToLevelName).not.toBeUndefined();
```

Comment:
no need for asserting that a function that you'll test exists.
you'd test that something exists when you don't actually know what it should do e.g. a mandatory callback function